fcp-0100.md

```diff
@@ -19,9 +19,9 @@ unnecessary friction.
 
 While it initially felt like similar to the difference between i486
 and i586, the number of changes required to make this happen are
-larger, especially in ports. Linux has agressively adopted armv7 for
-it's configuration, so having our uname -p match Linux will make the
-ports easier to adopt.
+larger, especially in ports. Linux has agressive adopted armv7 in its
```

Comment:
I think you actually meant 'agressively', or 'been agressive in'

Reply:
s/in its/as its/ ?

Reply:
yup fixed.